src/plugin/cesium/sync/featureconverter.js

```diff
@@ -540,11 +540,18 @@ plugin.cesium.sync.FeatureConverter.prototype.olLineStringGeometryToCesium = fun
  */
 plugin.cesium.sync.FeatureConverter.prototype.createLinePrimitive = function(positions, context, style, opt_type) {
   var type = opt_type || 'PolylineGeometry';
-  var appearance = new Cesium.PolylineColorAppearance();
 
   var width = this.extractLineWidthFromOlStyle(style);
   var color = this.extractColorFromOlStyle(style, true);
   color.alpha *= context.layer.getOpacity();
+  var lineDash = this.getDashPattern(style);
+
+  var appearance = new Cesium.PolylineMaterialAppearance({
+    material: Cesium.Material.fromType(Cesium.Material.PolylineDashType, {
```

Comment:
This should only be using `Cesium.Material.PolylineDashType` when there is actually a line dash. Otherwise it should use `Cesium.Material.ColorType`, which is more efficient to render because it doesn't have to consider a dash pattern. I would suggest adding a call like `getLineMaterial = function(options)`, with `color` and `dashPattern` in the options for now. That allows us to expand support in the future without API changes or a parameter train.

Reply:
So keep it as a `PolylineMaterialAppearance` and vary the material?

Reply:
Yep! Let me know if you have any issues. I tinkered with it quick, and it seemed to work as described.